.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: go
 go:
-  - 1.3
```

Comment:
Instead of mutexes I use atomic.Value in our test, this was not available in go1.3.